support-modules/bigquery/src/main/scala/com/gu/support/acquisitions/AcquisitionDataRowMapper.scala

```diff
@@ -14,9 +14,9 @@ object AcquisitionDataRowMapper {
       acquisition.promoCode.map("promo_code" -> _),
       acquisition.paymentProvider.map("payment_provider" -> paymentProviderName(_)),
       acquisition.printOptions.map(p => "print_options" -> Map(
-        "product" -> p.product,
+        "product" -> p.product.value,
         "delivery_country_code" -> p.deliveryCountry.alpha2
-      )),
+      ).asJava),
```

Comment:
Why does this need to be converted to a Java map?

Reply:
It's a Java sdk and that's what it requires unfortunately otherwise it fails silently 🤦